src/client/sandbox/event/message.js

```diff
@@ -40,8 +41,23 @@ export default class MessageSandbox extends SandboxBase {
         this.iframeInternalMsgQueue = [];
     }
 
+    _getMessageData (e) {
+        const rawData = nativeMethods.messageEventDataGetter && isMessageEvent(e)
+            ? nativeMethods.messageEventDataGetter.call(e)
+            : e.data;
+
+        return typeof rawData === 'string' ? parseJSON(rawData) : rawData;
+    }
+
+    _callOriginListener (originListener, e) {
```

Comment:
Move to util function (used in 2 places) and rename to `callEventListener` 
https://github.com/DevExpress/testcafe-hammerhead/blob/e4a4096b8d99dfb3a65333276f10f061589a6edf/src/client/sandbox/event/listeners.js#L74
https://github.com/DevExpress/testcafe-hammerhead/blob/cc241be3cfae15f31a0d9295f41a064f53810e39/src/client/sandbox/event/message.js#L69